app/controllers/v1/supplemental_claims/contestable_issues_controller.rb

```diff
@@ -3,6 +3,8 @@
 module V1
   module SupplementalClaims
     class ContestableIssuesController < AppealsBaseControllerV1
+      service_tag 'appeal-application'
```

Comment:
not sure if there's a rhyme or reason to the tags being assigned here, but this one feels a bit more vague than the other ones. any reason not to use `supplemental-claim` here (and below)?

Reply:
I had that question too, but benefits crew POs picked that name - it does have some alignment in context with other `*-application` service names, indicating "this is a veteran applying for a benefit", vs. viewing information.

Reply:
gotcha, makes sense. thanks for clarifying!